services/brig/index/src/Brig/Index/Options.hs

```diff
@@ -210,3 +234,12 @@ _IndexName = iso (\(ES.IndexName n) -> n) ES.IndexName
 
 _Keyspace :: Iso' C.Keyspace Text
 _Keyspace = iso C.unKeyspace C.Keyspace
+
+toESServer :: URIRef Absolute -> ES.Server
+toESServer =
+  ES.Server
+    . view utf8
+    . serializeURIRef'
+    . set pathL mempty
+    . set queryL mempty
+    . set fragmentL mempty
```

Comment:
I would much rather change our options to be `--elasticsearch-host` and `--elasticsearch-port` as those are the only things we care about and it would be aligned with how we read `CassandraSettings`. Should we add a `FUTUREWORK`?

Reply:
yes, good plan!